processor.go

```
@@ -50,11 +50,21 @@ func (dp *DefaultProcessor) execute(ctx context.Context, notifier Notifier, stat
 	}
 	url := "gs://" + bucket + "/" + name
 
+	service := &WatchService{ctx}
+	topic, err := service.topicFor(url)
+	if err != nil {
+		return err
+	}
+	if topic == "" {
+		log.Infof(ctx, "No topic found for %q", url)
+		return nil
```

At first I saw `return nil`, I thought it should return an error,
but I understand that it should not return an error to prevent OCN retry loop, is this correct?
(ref: https://cloud.google.com/storage/docs/object-change-notification#_Reliable_Delivery)

---

@minimum2scp 
> but I understand that it should not return an error to prevent OCN retry attempts, is this correct?

Yes, actually it aims only to prevent OCN retry attempts but also to ignore the uploaded file intentionally.

If an user uploads various files to GCS but they don't want to process files with specified extensions, 
`blocks-gcs-watcher` should ignore the files without that specified extensions. So I'm using`log.Infof` instead of `log.Errorf`.

For example, `blocks-gcs-watcher` ignores text files about images which are placed in the same directory of the images.

---

OK, thank you.